_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -573,10 +569,6 @@ void VideoDECODEVP9_HW::UpdateVideoParam(mfxVideoParam *par, VP9DecoderFrame con
     VM_ASSERT(par);
 
     MFX_VP9_Utility::FillVideoParam(m_core, frameInfo, par);
```

Comment:
Done. Added FillVideoParam(m_core, frameInfo, par) status check.

Reply:
I'd rather remove status from `MFX_VP9_Utility::FillVideoParam` it looks quite useless. Function semantic is format MFX parameters from domain specific headers. The latter ones must be valid at time when `FillVideoParam` is invoked, and actually they are not checked. The only thing that is checked by the `FillVideoParam` is its arguments - `MFX_CHECK_NULL_PTR2(core, params)`. This may be resolved by
1. Using of reference instead of pointer, or since `FillVideoParam` is internal we may use assertion for `mfxVideoParam*`.
2.  Pointer to core is needed only for checking if platform is HW and set `Shift` properly. We may pass platform type as argument.